app/config/local/app.php

```diff
@@ -26,7 +26,7 @@
 	|
 	*/
 
-	'url' => 'http://localhost/atlas',
+	'url' => 'http://openmrsatlas.local',
```

Comment:
I didnt see this value getting used anywhere. Still it would be better to keep it unchanged.  Shall I change this to previous value and create a pull request again?